src/basic/options.fs

```diff
@@ -187,7 +187,15 @@ let get_include_path (dirname:string) =
     !_include_path@["."; h ^ "/lib"; h ^ "/lib/fstar"])
 
 let prims () = match !prims_ref with
-  | None -> Util.format1 "%s/lib/prims.fst" (get_fstar_home ())
+  | None -> 
+    (let filen = "prims.fst" in
```

Comment:
Oh interesting. That would've worked in OCaml. I wasn't aware that the parsing was different in this specific case between F# and OCaml.